test/codec_switching/codec_switching_integration.js

```diff
@@ -35,6 +37,10 @@ describe('Codec Switching', () => {
     eventManager = new shaka.util.EventManager();
     waiter = new shaka.test.Waiter(eventManager);
     waiter.setPlayer(player);
+
```

Comment:
@tykus160 can you add `player.configure('streaming.allowMediaSourceRecoveries', false);`?

I think this is going to bloom more errors... which right now are automatically corrected....